### PR TITLE
Update ug and website layout

### PR DESCRIPTION
**Pull request by revdrag**
- Updated website layout
  - Remove tutorials
  - Added Appendix - Planned Enhancements
- Update UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -179,11 +179,11 @@ Most commands in EduTrack require various parameters. You can find their constra
...
+| MEMO, PARTICIPATION | A string consisting of any characters.                                                                                                                                                                                                              |
 | SCHEDULE            | A string that follows the format `day, HH:MM-HH:MM`: <br> - `day` can be one of the following options: "mon","tue","wed","thu","fri" <br> - `HH:MM-HH:MM` signifies a 24-hour clock format, where the start time must be earlier than the end time. |
-| ID                  | A string consists of alphanumeric characters, and it should begin with "A" and end with a capital letter (A to Z).                                                                                                                                  |
+| ID                  | A string that starts with "A", followed by only numerical characters (0-9) and end with a single capitalised alphabet (A-Z).                                                                                                                        |
```
Comment (DonovanJJ) [inline docs/UserGuide.md]: Slight grammatical  error here, "and ~~ends~~ end with a single..."

**Review by DonovanJJ — APPROVED** [review]
LGTM 👍 
Just one slight grammatical error. 
Thanks for helping standardise the UG!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -678,21 +679,21 @@ Edits an existing student from your specified class. You can edit the student na
 **Command Format:** `edit /s STUDENT_INDEX /c CLASS_NAME [/n STUDENT_NAME] [/id ID] [/m MEMO] [/p PARTICIPATION]`
 
 **Examples:**
-* `edit /s 1 /c CS2103T /n Bob`: Edits the name of the first student in the class CS2103T to "Bob".
-* `edit /s 2 /c CS2103T /id A0231234U`: Edits the id of the second student in the class CS2103T to "A0231234U".
+* `edit /s 1 /c CS2103T /n Bob` : Edits the name of the first student in the class CS2103T to "Bob".
+* `edit /s 2 /c CS2103T /id A0231234U` : Edits the id of the second student in the class CS2103T to "A0231234U".
 
 **Sample Usage:**
```
Comment (songgthu) [inline docs/UserGuide.md:699]: I think you can add screenshots for this feature to illustrate the sample usage better

**Review by songgthu — APPROVED** [review]
LGTM!

**Pull request merged**